ovos_PHAL_plugin_commonIOT/device_manager.py

```diff
@@ -47,60 +47,56 @@ def __init__(self, bus):
     def handle_get_brightness(self, message):
         device = self.search_for_device(message.data.get("device"))
         if device:
-            return device.brightness
-        return None
+            msg = message.response(data={"brightness": device.brightness})
+            self.bus.emit(msg)
 
     def handle_set_brightness(self, message):
         device = self.search_for_device(message.data.get("device"))
         if device:
             try:
-                device.set_brightness(message.data.get("brightness"))
-                return device.brightness
+                msg = message.response(data={"brightness": device.change_brightness(message.data.get("brightness"))})
             except Exception as e:
                 LOG.error(e)
-                return e
-        return False
+                msg = message.response(data={"brightness": e})
```

Comment:
data here should be `"error": e`, it's not a valid brightness value so it should use a different key